.github/workflows/docs-link-check.yml

```diff
@@ -0,0 +1,62 @@
+name: Documentation Link Check
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
Suggest some random hour / minutes: It probably doesn't matter, but if everyone puts `0 0` for everything everywhere might cause small spikes.